README.md

```diff
@@ -1635,6 +1641,78 @@ The bridge-domain ID. Valid values are one or range of integers.
 The Virtual Network Identifier (VNI) id that is mapped to the VLAN. Valid values are one or range of integers
 
 --
+### Type: cisco_dhcp_relay_global
+
+Manages configuration of a DHCP relay global configuration.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I3(1)        | 1.4.0                  |

```

Comment:
Based on the test results, the `OS Minimum Version` should be `7.0.3.I2.2e` right?